### PR TITLE
Replaced dead [inventory file for Ansible] link

### PR DESCRIPTION
**Pull request by dtaivpp**
Removed bad link for Ansible Inventory guide and replaced with the current link.

### CODE REVIEW

**Comment by dtaivpp** [comment]
I signed it

**Review by Arhell — COMMENTED** [review]
more info about CLA, check please
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by dtaivpp** [comment]
I signed it

File: content/en/docs/setup/production-environment/tools/kubespray.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Kubespray provides the following utilities to help provision your environment:
 
 ### (2/5) Compose an inventory file
 
-After you provision your servers, create an [inventory file for Ansible](https://docs.ansible.com/ansible/intro_inventory.html). You can do this manually or via a dynamic inventory script. For more information, see "[Building your own inventory](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#building-your-own-inventory)".
+After you provision your servers, create an [inventory file for Ansible](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html). You can do this manually or via a dynamic inventory script. For more information, see "[Building your own inventory](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#building-your-own-inventory)".
```
Comment (kbhawkey) [inline content/en/docs/setup/production-environment/tools/kubespray.md:53]: Hello @dtaivpp .
Does this link change represent the same change as PR #27473?

Comment (dtaivpp) [reply content/en/docs/setup/production-environment/tools/kubespray.md:53]: Yup! Feel free to close this one and merge that one if they don't need to rebase as well

**Comment by kbhawkey** [comment]
Thanks @dtaivpp .
/close

**Pull request closed without merging**